doc/tutorials/snippet-sharing-rest-service.md

```diff
@@ -146,14 +161,21 @@ configuration:
 > kdb set system/sw/elektra/restbackend/#0/current/cppcms/http/script_names/#0 "/"
 ```
 
+Note: here we have not used the option `-N system` because the CppCMS configuration is
```

Comment:
Any reason to do here differently? Maybe validation will be added later? Or we should always do validation (even when setting per namespace).